crates/kornia-tensor/src/tensor.rs

```diff
@@ -1374,6 +1397,15 @@ mod tests {
         Ok(())
     }
 
+    #[test]
+    fn from_raw_parts() -> Result<(), TensorError> {
+        let data: Vec<u8> = vec![1, 2, 3, 4];
+        let t = Tensor::from_raw_parts([2, 2], data.as_ptr(), data.len(), CpuAllocator)?;
+        assert_eq!(t.shape, [2, 2]);
+        assert_eq!(t.as_slice(), &[1, 2, 3, 4]);
+        Ok(())
+    }
```

Comment:
The Tensor is freed up here, then data_vec then the same underlying buffer is freed up

Reply:
fixed with forget